src/components/activities/ActivityCard.jsx

```diff
@@ -7,6 +7,13 @@ import PropType from 'prop-types';
  * @extends React.Component
 */
 class ActivityCard extends Component {
+  static propTypes = {
```

Comment:
You omitted comments

Reply:
yeah just copy<>paste the comments @john555 had already added